src/platform/macos.rs

```diff
@@ -56,6 +56,14 @@ pub trait WindowExtMacOS {
     /// And allows the user to have a fullscreen window without using another
     /// space or taking control over the entire monitor.
     fn set_simple_fullscreen(&self, fullscreen: bool) -> bool;
+
+    /// Toggles the `Option` key being interpretted as an `Alt` modifier.
+    ///
+    /// This will ignore diacritical marks and accent characters from
+    /// being processed as received characters. Instead, the input
+    /// device's raw character will be placed in event queues with the
+    /// Alt modifier set.
+    fn set_option_as_alt(&self, option_as_alt: bool);
```

Comment:
Why was the `with_option_as_alt` configuration removed? I was suggesting both since that seems to be the most common scenario for features like these.

Reply:
I must have misunderstood this comment? https://github.com/rust-windowing/winit/pull/1449#issuecomment-584730213

You would like the ability for WindowBuilder and the configuration to modify this behavior? Okay, I can bring that back.

Reply:
Yes. You want to be able to do it at startup, but also modify it later at any time. So we need both.